User-Help/Voting.md

```diff
@@ -0,0 +1,40 @@
+| URL slug | Category |
+|:--------:|:--------:|
+| voting | Guidance |
+
+# Why should I vote?
+
+Voting is the primary indicator of valuable content.  Vote up posts that add to the community, and vote down ones that detract from it.  (Many posts do neither; you don't have to vote on everything.)
+
+Voting influences a post's score.  Post score contributes to earning abilities.  When you vote, you are, in a sense, saying "I would like the community to get more (or less) of this kind of post".
+
+
+# When should I vote?
+
+![voting buttons](/assets/voting.png]
+
+You can vote up or down on posts by clicking the arrows next to the post.  In general, vote up content that is clear, interesting, well-supported, and, for answers, correct.  Vote down content that is unclear or demonstrably wrong.
+
+**Voting is not a popularity contest.**  Don't vote based on who posted something -- don't vote for your friends or vote down posts from people you don't like.  
```

Comment:
We should discuss em vs en dashes, double en is probably not ideal?

Reply:
Some of them should have been other punctuation (fixed).  I replaced the remaining ones with &emdash;, which I think works in Markdown?